controllers/v1/cloud-services/files.js

```diff
@@ -183,6 +183,88 @@ module.exports = class Files {
         })
 
     }
+    // Internal API for Data Team As part of CSP implementation
+    /**
+     * @api {post} /kendra/api/v1/cloud-services/files/bucketSpecificUrl?urlType=presignedUrl/downloadableUrl
```

Comment:
@VISHNUDAS-tunerlabs Changes as below - 

URL - /kendra/api/v1/cloud-services/files/actions

Body - 
{
            "request" : {     
                "files": ["uploadFile.jpg", "uploadFile2.jpg"]
             },
            "action" : "signedUrl",
            "operation"  : "read/write"
            "ref" : "datateam",
            "bucket" : "sample_bucket_name",
            "expiresIn": 262500 //min
        }

Reply:
@aks30 change will be
URL - /kendra/api/v1/cloud-services/storage/operations

Body -
{
"request" : {
"files": ["uploadFile.jpg", "uploadFile2.jpg"]
},
"action" : "signedUrl",
"operation" : "read/write"
"ref" : "datateam",
"bucket" : "sample_bucket_name",
"expiresIn": 262500 //sec
}